src/silvimetric/resources/metric.py

```diff
@@ -1,20 +1,19 @@
 import json
 import numpy as np
-from typing import Callable, Optional, Any, Union
-from scipy import stats
+from typing import Callable, Optional, Any, Union, List
 from inspect import getsource
 from tiledb import Attr
 import dask
 import base64
-import pickle
 import dill
+import pandas as pd
```

Comment:
Were project dependency requirements updated to add Pandas?

Reply:
I don't believe so, but should probably put it in the environment file anyway for good measure